Adding files for Range Dome Analysis

topowright opened the pull request:
Adding files for Range Dome Analysis. 

Added: Toolboxes, Common Data for symbology rules from city engine and data examples for the tools to run properly.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/129342386639048/187727964630776)

mfunk commented:
Going through the Range Dome PR and got a few things that aren’t looking good for these tools. I know you didn't create them, but between the two of us we need to figure this out before we can merge them. I'm not confident this is a time where we can just merge it as-is and fix it later. In general the stuff here doesn't seem to match what I'm reading on the Solutions Page (http://solutions.arcgis.com/defense/templates/range-dome-analysis/).

1. There are two toolboxes, the names are funny and don't match the others, and one of them has no tools. Also neither of the toolbox names matches what is on the Solutions page.
![image](https://cloud.githubusercontent.com/assets/3093182/18956618/e3377dca-8610-11e6-9856-3bed3e1d303a.png)

My recommendation here is to kill the empty toolbox and keep the existing one (with name change) to match the other tools in the repo, and in the ArcGIS platform.

2. The one tool in *RangeDomePyProTools.pyt* is **Detect non-closed volumes** (not creating range domes?) and does't have any tool help (the associated XML has header info but no content). Looking at the Solutions page isn't there a tool to create Range Dome Corridor, 

My recommendation here is much harder. We need documentation for this tool. We cannot have tools without doc.

3. There are two geodatabases *RangeDomeAnalysis.gdb* which is empty, and *rangeoutput.gdb* which has a bunch of features in it. It seems that *rangeoutput.gdb* is data output from the tools used in the template which doesn't server a purpose here. The *tooldata* stuff is where data the tool requires to run is stored. This also doesn't quite match what is listed in the Solutions page.

Again here, my recommendation is to drop the tooldata.

topowright commented:
@mfunk 
Addressing your issues: 
1.
Just added the tools that were missing and changed the name of the toolboxes to include the PYT toolbox. 

2. 
This is a relic of the old template that I am not sure has much use. This has been deleted.

mfunk commented:
Closing this one without merging as Range Dome Analysis is being deprecated.
